### DISCUSSION TITLE
Making the LDK Dev Meeting Minutes available

### DISCUSSION THREAD

**Discussion by ariard** [lightningdevkit/rust-lightning#1608] [Ideas / Feature Requests]
"Could be interesting to have a Meeting subsection on the website where we would log past meeting minutes to ease newcomers catching up with LDK context. Rn, i think minutes are spread over the GH wiki and google docs and elsewhere".

**Comment by ConorOkus** [comment]
How about a combination of GitHub wiki + Hack.MD? I like the way the Fedimint guys have organised things - https://github.com/fedimint/minimint/wiki

**Comment by ariard** [comment]
Yes a Github wiki sounds really nice to me.
Note, I think it's good to be "liberal" with the editorial policy of the Wiki to invite contributors to provide contents, even if not completely structured way and keep the website for the hardened, foolproofed content. We're likely going to have a trade-off between content quality and content scarcity. 

FYI, here the old Meeting notes : https://github.com/lightningdevkit/rust-lightning/wiki/Meetings. After that there is a black hole then Val has taken Google Docs.

**Reply by tnull** [reply]
Hm, maybe it would be easiest if we'd just get into the habit of copy/pasting the final notes collected in Google Docs during the meeting to the 'old' link? 

@valentinewallace Any thoughts? If you'd rather not have another task, I can sign responsible for that?

**Comment by tnull** [comment]
I now restructured the old [Meetings Wiki page](https://github.com/lightningdevkit/rust-lightning/wiki/Meeting-Notes) and will keep it updated regularly.
